src/bin/huak/commands/add.rs

```diff
@@ -15,7 +16,7 @@ pub fn cmd() -> Command<'static> {
 
 /// Run the `add` subcommand.
 pub fn run(args: &ArgMatches) -> CliResult<()> {
-    let _ = args.get_one::<String>("dependency");
-
-    unimplemented!()
+    let dependency = args.get_one::<String>("dependency");
+    add_project_dependency(dependency.unwrap().to_string(), false)?;
```

Comment:
I'll checkout add_project_dependency, but can we use the same signature (or roughly) that we use for ops::remove::remove_project_dependency?

Reply:
so
```suggestion
    if let Err(e) = ops::add_project_dependency(&project, dependency) {
        return Err(CliError::new(e, ExitCode::FAILURE));
    }
```